packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.5
+
+- Avoid using push() to push the same page in example, since is not supported.
+  [#105150](https://github.com/flutter/flutter/issues/105150)
```

Comment:
Is this intentionally on the next line? Will it cause any formatting issues? It looks like all other versions put the issue on the same line as the message